Cargo.toml

```diff
@@ -62,6 +62,11 @@ zip = { version = "0.6.2", default-features = false, features = [ "deflate", "bz
 # Enable feature zstdmt to enable multithreading in libzstd.
 zstd = { version = "0.10.0", features = [ "bindgen", "zstdmt" ], default-features = false }
 
+arrayvec = "0.7.2"
```

Comment:
Nitpick: put this one in the bulk of the list, in alpha order

Reply:
Done!